pkg/kubelet/rkt/rkt.go

```diff
@@ -935,6 +942,12 @@ func (r *Runtime) preparePod(pod *api.Pod, pullSecrets []api.Secret) (string, *k
 		newUnitOption("Service", "KillMode", "mixed"),
 	}
 
+	if pod.Spec.SecurityContext != nil && pod.Spec.SecurityContext.SELinuxOptions != nil {
+		opt := pod.Spec.SecurityContext.SELinuxOptions
+		selinuxContext := fmt.Sprintf("%s:%s:%s:%s", opt.User, opt.Role, opt.Type, opt.Level)
```

Comment:
Sorry for the late response.  I had meant that if you don't specify all of the selinux context, you shouldn't get a malformed one.  What we do elsewhere is to take the specified fields of selinux context and overlay them onto the selinux context of the kubelet root dir.  So, if root dir has `system_u:object_r:svirt_sandbox_file_t:s0` and the user specifies only `s1,c2`, they should get: `system_u:object_r:svirt_sandbox_file_t:s1,c2`.